sdk/storage/Azure.Storage.Blobs/src/Models/UploadContent.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using Metadata = System.Collections.Generic.IDictionary<string, string>;
+
+namespace Azure.Storage.Blobs.Models
+{
+    /// <summary>
+    /// Blob content to be uploaded by a <see cref="Specialized.BlobBaseClient"/>.
+    /// </summary>
+    public class UploadContent
```

Comment:
We should probably name this `BlobUploadContent` because we've been burned a couple of times when we haven't put the `Blob` prefix in front of things that we eventually want to add to Queues/Files/DataLake down the road.